### PR TITLE
[NTUSER] Fix PeekMessageA for MsgFilterLow/High mouse clicks

### PR DESCRIPTION
**Pull request by Doug-Lyons**
## Purpose

_Fix MSO and Word Viewer not closing when "X" on title bar is clicked._
_This is another @I_Kill_Bugs patch._

JIRA issue: [CORE-14436](https://jira.reactos.org/browse/CORE-14436)
JIRA issue: [CORE-16985](https://jira.reactos.org/browse/CORE-16985)

## Proposed changes

_Fix MsgFilterLow/MsgFilterHigh handling for nonclient areas for mouse events._

### CODE REVIEW

**Review by JoachimHenze — APPROVED** [review]
My approval is based on the testbot results
VBox: https://reactos.org/testman/compare.php?ids=93128,93131 LGTM
KVM: https://reactos.org/testman/compare.php?ids=93129,93132 LGTM

File: win32ss/user/ntuser/msgqueue.c [hunk 1]
```diff
@@ -2000,6 +1999,13 @@ co_MsqPeekHardwareMessage(IN PTHREADINFO pti,
 
          UpdateKeyStateFromMsg(MessageQueue, &msg);
          AcceptMessage = co_IntProcessHardwareMessage(&msg, &Remove, &NotForUs, ExtraInfo, MsgFilterLow, MsgFilterHigh);
+         
+         if (MsgFilterLow != 0 || MsgFilterHigh != 0)
+         {
+             /* Don't return message if not in range. */
```
Comment (HBelusca) [inline win32ss/user/ntuser/msgqueue.c:2005]:
```suggestion
             /* Don't return message if not in range */
```

**Review by HBelusca — APPROVED** [review]

File: win32ss/user/ntuser/msgqueue.c (hunk 1 above)
Comment (Doug-Lyons) [reply win32ss/user/ntuser/msgqueue.c]: Fixed. Thanks.

**Comment by Doug-Lyons** [comment]
Thanks @HBelusca for your approval and commit here and @JoachimHenze for your approval.

**Pull request merged**